charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -98,23 +98,23 @@ spec:
         readinessProbe:
           httpGet:
             port: 8444
-            path: /healthz
+            path: /healthz/ready
             scheme: HTTPS
           failureThreshold: 1
-          initialDelaySeconds: 20
+          initialDelaySeconds: 40
           periodSeconds: 10
           successThreshold: 1
-          timeoutSeconds: 2
+          timeoutSeconds: 5
         livenessProbe:
           httpGet:
             port: 8444
             path: /healthz
             scheme: HTTPS
           failureThreshold: 3
-          initialDelaySeconds: 20
+          initialDelaySeconds: 40
```

Comment:
In contrast to readiness probes, this is ok, to prevent the API Server from being restarted too soon.